docs/source/installation-guide/install-kubernetes.md

```diff
@@ -16,6 +23,28 @@ The Kubernetes deployment of the Gluu Server, also called Cloud Native (CN) Edit
 - [Passport interactions diagram](#architectural-diagram-of-oxpassport)
 - [Jackrabbit interactions diagram](#working-with-jackrabbit)
 
+## Requirmentes for accessing docker images and asset
+
+1. Contact sales@gluu.org for credentials to access and pull our docker images. Existing customers should have recieved the credentials already. If you are using [`pygluu-kubernetes.pyz`](https://github.com/GluuFederation/cloud-native-edition/releases) the tool that parses values.yaml you can skip the next steps. We recommend using helm manually.
+2. Follow the [guide](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-secret-by-providing-credentials-on-the-command-line) to create a secret in your kubernetes environment holding the docker credentials.
```

Comment:
We need to mention that when creating the secrets for docker credentials, the same namespace should be used.